langfuse-core/src/utils.ts

```diff
@@ -90,3 +92,48 @@ export function getEnv<T = string>(key: string): T | undefined {
   }
   return;
 }
+
+interface Params extends LangfuseCoreOptions {
+  publicKey?: string;
+  secretKey?: string;
+}
+
+export function configLangfuseSDK(params?: Params, secretRequired: boolean = true): Params {
+  if (!params) {
+    params = {};
+  }
```

Comment:
use params: Params = {} instead?

Reply:
Not necessary. It is of that type already.